features/attachments.feature

```diff
@@ -25,8 +25,8 @@ Feature: Attachments
       """
     When I run cucumber-js
     Then scenario "some scenario" "Before" hook has the attachments:
-      | DATA     | MEDIA TYPE |
-      | iVBORw== | image/png  |
+      | DATA     | MEDIA CONTENT TYPE | MEDIA ENCODING |
```

Comment:
Let's just use "media type" - see https://stackoverflow.com/questions/9258108/whats-the-difference-between-mediatype-contenttype-and-mimetype

Reply:
Updated